### ISSUE TITLE
Cannot find bower_components/drop/drop.min.js

### ISSUE THREAD

**Issue by ahx** [shipshapecode/ember-drop#7]
I am not sure if this is an issue with [drop](https://github.com/HubSpot/drop/) or ember-drop.

When trying to use the plugin with a new Ember project, ember-cli cannot find drop.min.js.  Here's what happens:
```
~/Desktop/foo[master*]: ember install ember-drop && ember s
Installed packages for tooling via npm.
installing ember-drop
  install bower package drop
  cached https://github.com/HubSpot/drop.git#1.4.2
  cached https://github.com/HubSpot/tether.git#1.2.0
Installed browser packages via Bower.
Installed addon package.
Livereload server on http://localhost:49152
Serving on http://localhost:4200/
The Broccoli Plugin: [SourceMapConcat: Concat: Vendor /assets/vendor.js] failed with:
Error: ENOENT: no such file or directory, stat '/Users/ahaller/Desktop/foo/tmp/source_map_concat-input_base_path-4DIIyKHU.tmp/0/bower_components/drop/drop.min.js'
```
I noticed that there is `bower_components/drop/src/drop.js`, but no drop.min.js.

**Comment by ahx** [comment]
Alright, this seems to be fixed by https://github.com/shipshapecode/ember-drop/commit/d41bbb611e052ef8cb2d6fb27d7762fd39e907dc. Sorry for the noise.

**Comment by rwwagner90** [comment]
No problem @ahx. I haven't done much with this addon in awhile. Do you have any feature requests?

**Comment by lukeclewlow** [comment]
I still have this problem, how can I get code for after that commit as a dependency on my Ember project?

**Comment by rwwagner90** [comment]
Sorry guys, had no idea I hadn't actually published any of the updates. I just published 0.0.5, so you should be able to just update to that version @lukeclewlow.
